lib/screens/home_screen/overview/widgets/mini_calendar.dart

```diff
@@ -73,10 +69,7 @@ class MiniCalendar extends StatelessWidget {
               bottomLeft: Radius.circular(5),
               bottomRight: Radius.circular(5),
             ),
-            border: Border.all(
-              color: theme.darkGreen1!,
-              width: 2,
-            ),
+            border: Border.all(color: theme.darkGreen1!, width: 2),
```

Comment:
Well, you deleted the trailing comma and hence we have a different code format here :) Which one should we use?